features/provider/admin/webhooks/new.feature

```diff
@@ -0,0 +1,40 @@
+@javascript @webhook
+Feature: Provider webhooks
+
+  As a provider, I want to configure the webhook I have previously created.
+
+  Background:
+    Given a provider is logged in
+    And the provider has "web_hooks" switch allowed
+
+  Scenario: Navigation
+    Given they go to the provider dashboard
+    When they select "Account Settings" from the context selector
+    And press "Integrate" within the main menu
+    And follow "Webhooks" within the main menu
+    Then the current page is the new webhook page
+
+  Scenario: Creating a webhook
+    Given they go to the new webhook page
+    And there shouldn't be a button to "Ping!"
+    When the form is submitted with:
+      | URL | http://google.com |
```

Comment:
I don't get why we are using `google.com` in our examples :upside_down_face: maybe it needs to be changed to something more redhatt-y or 3scale-ey

Reply:
Ah, yes! In fact I meant to change it, but cucumbers sucked all my energy and I forgot. I will do it asap.